### ISSUE TITLE
Very small mAP values with train.py - Cityscapes to Foggy Cityscapes

### ISSUE THREAD

**Issue by AmrinKareem** [giuliomattolin/ConfMix#7]
Hi @giuliomattolin, thank you so much for your work. 
I am facing the same issue as @HDUyiming - train.py results in very small mAP values - 

I have updated the Cityscapes2Foggy.yaml file with the locations of the datasets. Could you please share how you solved the issue? 

Here is a screenshot of my epochs: Thank you in advance :)

![image](https://user-images.githubusercontent.com/43820644/232917718-be8145b7-d64b-4444-92ba-a2dd85575924.png)

_Originally posted by @AmrinKareem in https://github.com/giuliomattolin/ConfMix/issues/3#issuecomment-1513856097_

**Comment by giuliomattolin** [comment]
Hi @AmrinKareem,

If you are running the Cityscapes2Foggy benchmark, make sure to use only images with $\beta$ = 0.02, **both** in the training and validation sets. From the screenshot, you posted I see that there are 1500 images in the validation set, while there should be only 500, which is the amount of validation images having $\beta$ = 0.02.

I hope this will help you!

**Comment by AmrinKareem** [comment]
Thank you so much, @giuliomattolin. that was the problem exactly. Although I couldn't reproduce the exact values, I was able to match it in most cases. Thank you for your prompt help!
